Userland/Libraries/LibWeb/Page/Page.h

```diff
@@ -152,7 +152,7 @@ class PageClient {
     virtual Gfx::IntRect page_did_request_maximize_window() { return {}; }
     virtual Gfx::IntRect page_did_request_minimize_window() { return {}; }
     virtual Gfx::IntRect page_did_request_fullscreen_window() { return {}; }
-    virtual void page_did_start_loading(const AK::URL&) { }
+    virtual void page_did_start_loading(const AK::URL&, bool) { }
```

Comment:
If you keep this as a bool it should definitely have a name in the declaration.
I don't think a bool is self-evident.

Alternatively you can add a new enum something like PageLoadIsRedirect::{Yes, No}